### DISCUSSION TITLE
关于导出为markdown，如何去除一些不必要的信息

### DISCUSSION THREAD

**Discussion by Lusenc** [windingwind/zotero-better-notes#529] [Q&A]
如下图所示，如何只保留提取的关键文本？
![image](https://github.com/windingwind/zotero-better-notes/assets/50198715/27ac7588-aba2-4cb7-891d-852fcc63c2e6)

**Comment by youngfish42** [comment]
应该需要修改对应的导出模板

**Reply by Yehenara-ZZ** [reply]
请问怎么修改变成这种zotero://open-pdf/library/items/ 链接格式
![图片1](https://github.com/windingwind/zotero-better-notes/assets/83431060/f4b08b28-446e-4925-b5de-2c2ee4340f42)
![图片3](https://github.com/windingwind/zotero-better-notes/assets/83431060/d8441e23-cbd9-4386-aa1f-f2e6f145bb22)
